purchasing/settings.py

```diff
@@ -64,7 +64,7 @@ class TestConfig(Config):
     BROWSERID_URL = 'test'
     ASSETS_DEBUG = True
     CITY_DOMAIN = 'foo.com'
-    MAIL_SUPPRESS_SEND = True
+    MAIL_SUPPRESS_SEND = False
```

Comment:
This should be `True` in test config, otherwise you are going to try to send emails during your tests (which is why the tests are erroring on Travis :) )